tasks/main.yml

```diff
@@ -1,36 +1,107 @@
 ---
-- name: Ensure directory exists
+- name: Ensure group for backups user exists
+  group:
+    name: "{{ backups_role_user_group }}"
+    state: present
+
+- name: Ensure restricted user for backups exists
+  user:
+    name: "{{ backups_role_user_name }}"
+    group: "{{ backups_role_user_group }}"
+    system: true
+    state: present
+  when: backups_role_user_name != 'root'
+
+
```

Comment:
Extra line

Reply:
Adressed in 0c4a8dbbea5ca9bf73328374b0f1210ecf5213c1